src/App.css

```diff
@@ -1,38 +1,97 @@
+
+.clearfix:after {visibility: hidden; display: block; font-size: 0; content: ''; clear: both; height: 0; }
+
+html { -moz-box-sizing: border-box; -webkit-box-sizing: border-box; box-sizing: border-box;}
+
+*, *:before, *:after {box-sizing: inherit}
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+
+
+body {
   text-align: center;
+  background-color: #d8bae3;
+  font-family:Arial, Helvetica, sans-serif;
+  font-weight: 600;
+}
+
+header {
+  padding: 30px 0 15px;
+}
+
+form {
+  font-size: 1.2rem;
+  padding-bottom: 100px;
+}
+
+p {
+  font-size:1.1rem;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+h1 {
+  font-size: 3rem;
+  text-transform: uppercase;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.recipes {
+    display: grid;
+    grid-template-columns: repeat(2,1fr);
+    gap: 50px;
+    margin: 50px;
+    justify-content: center;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.recipe-card {
+    background: #f8f8f8;
+    box-shadow:0 0 5px 17px #a272b3;
+    display: flex;
```

Comment:
I really like how you displayed your recipe cards on the page. I allows the user to see two recipes on the page at one, which I find valuable. I might change mine to do the same.